### ISSUE TITLE
Properties.overrideParameters doesn't do anything?

### ISSUE THREAD

**Issue by marcsaegesser** [typelevel/scalacheck#221]
Maybe I'm missing something but the `Properties.overrideParameters` method doesn't seem to actually do anything. The comment says it's supposed to override the test parameters for current Properties class but all it seems to do is return the value it's passed and it doesn't have any impact on the test parameters used by the tests.

``` scala
/**
 * Changes to the test parameters that are specific to this class.
 * Can be used to set custom parameter values for this test.
 */
def overrideParameters(p: Test.Parameters): Test.Parameters = p
```

**Comment by rickynils** [comment]
I think the idea is that you override that function within your `Properties` instance, and return a modified version of the parameters that will be used when running the tests (for that property collection). The scaladocs are of course not telling you this :) So we can keep this issue open until the docs are fixed.

**Comment by marcsaegesser** [comment]
Got it. Thanks.

**Comment by ashawley** [comment]
Closing this issue.  The documentation was fixed in #524.  Overriding parameters was also fixed in #522.
